config/babel-preset-ibm-cloud-cognitive/index.js

```diff
@@ -1,37 +1,34 @@
 /**
- * Copyright IBM Corp. 2020, 2021
+ * Copyright IBM Corp. 2016, 2024
```

Comment:
```suggestion
 * Copyright IBM Corp. 2020, 2024
```

Was this intentionally changed?

Reply:
No, this was just accidental 😅